### PR TITLE
Balaclava face mask

### PR DESCRIPTION
**Pull request by JoeHammad1844**
## About the PR

Adds a balaclava face mask that blocks your identity, available in hacked clothesmates, winterdrobes and that theater one. also available in the SecTec and syndie drobe. they also spawn in maints lockers

## Why / Balance

would be interesting for traitorous roleplay
## Media

image
image
- [x] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

**Changelog**

:cl: JoeHammad
- add: Centcom has recently gotten a large shipment of balaclavas from space russia, and has dumped the excess all over the station

### CODE REVIEW

**Comment by JoeHammad1844** [comment]
looks like I did an oopsie

**Comment by VektorZ1** [comment]
I think it's also worth adding it to the aplink for 1 TC.

**Comment by JoeHammad1844** [comment]
eh I don't people to get people arrested just for wearing it

**Comment by NancokPS2** [comment]
Well, atm you need an emag to get one, so they are getting arrested regardless.

**Comment by basyachok** [comment]
it would be good to have some in the sec wardrobe

**Comment by JoeHammad1844** [comment]
> Well, atm you need an emag to get one, so they are getting arrested regardless.

no, you dont, you just snip the manager wire with wire cutters

**Comment by JoeHammad1844** [comment]
> it would be good to have some in the sec wardrobe

sure, why not

**Comment by JoeHammad1844** [comment]
I think I will also add them to maints loot pool

**Comment by NancokPS2** [comment]
> 

oh my bad, tought that wasn't implemented yet.

**Comment by JoeHammad1844** [comment]
I would make lizard sprites but I can't find the right images in the files

**Comment by JoeHammad1844** [comment]
I put the balaclavas into the sectec :godo:

**Comment by JoeHammad1844** [comment]
ill make a lizard sprite then ill open it.. when I get around to it

**Comment by JoeHammad1844** [comment]
i think it will be worth t to open it

File: Resources/Prototypes/Catalog/Fills/Lockers/misc.yml [hunk 1]
```diff
@@ -135,6 +136,8 @@
           prob: 0.20
         - id: DrinkSpaceLube
           prob: 0.20
+        - id: ClothingMaskBalaclava
```
Comment (Partmedia) [inline Resources/Prototypes/Catalog/Fills/Lockers/misc.yml:139]: I don't think this should be available in maints loot.

File: Resources/Prototypes/Catalog/Fills/Lockers/misc.yml [hunk 2]
```diff
@@ -218,6 +221,8 @@
           prob: 0.20
         - id: ModularReceiver
           prob: 0.1
+        - id: ClothingMaskBalaclava
```
Comment (Partmedia) [inline Resources/Prototypes/Catalog/Fills/Lockers/misc.yml:224]: Or here

**Review by Partmedia — CHANGES_REQUESTED** [review]
I think this is reasonable. However, I don't think that this should be available as regular maintenance loot, and instead only to syndicates and via emagged vendors. It should definitely not be available from the security vendor.

File: Resources/Prototypes/Catalog/Fills/Lockers/misc.yml (hunk 1 above)
Comment (JoeHammad1844) [reply Resources/Prototypes/Catalog/Fills/Lockers/misc.yml:139]: i dont want people to be valid just for wearing it or even have it be emag only, its intended to be a somewhat abundant and free clothing item just to conceal your identity and for space russian roleplay

**Comment by CatMagic1** [comment]
I don't see why it shouldn't be in maints loot. This is cool and I want it as a non-syndicate, even if it makes me suspicious.

**Comment by Nairodian** [comment]
I'm fine with it being maint loot. But don't put it in lockers, just doesn't fit. We already have random item map spawners for stuff like this.
`\Resources\Prototypes\Entities\Markers\Spawners\Random\maintenance.yml`
I'd rather it in the clothing or maybe even the scrap one.

**Comment by JoeHammad1844** [comment]
> I'm fine with it being maint loot. But don't put it in lockers, just doesn't fit. We already have random item map spawners for stuff like this. `\Resources\Prototypes\Entities\Markers\Spawners\Random\maintenance.yml` I'd rather it in the clothing or maybe even the scrap one.

good idea

**Comment by CatMagic1** [comment]
Nooo
This was really cool....

**Pull request closed without merging**